wire-ios-sync-engine/Source/Calling/WireCallCenterV3+MLS.swift

```diff
@@ -96,8 +96,16 @@ extension WireCallCenterV3 {
     // Leaves the possibles subconversation for the mls conference.
 
     private func leaveStaleConferenceIfNeeded(conversationID: AVSIdentifier) {
+        guard let viewContext = uiMOC,
+              let conversation = ZMConversation.fetch(
+                with: conversationID.identifier,
+                domain: conversationID.domain,
+                in: viewContext),
+              conversation.conversationType == .group else {
+            return
+        }
+
```

Comment:
To be consistent with the next guard:

```suggestion
        guard
            let viewContext = uiMOC,
            let conversation = ZMConversation.fetch(
                with: conversationID.identifier,
                domain: conversationID.domain,
                in: viewContext
            ),
            conversation.conversationType == .group
        else {
            return
        }

```